lab11-amanda/route/note-router.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const Note = require('../model/note.js');
+const createError = require('http-errors');
+
+let noteRouter = module.exports = new Router();
+
+//POST
+noteRouter.post('/api/notes', jsonParser, (req, res, next) => {
+  if(!req.body){
+    return next(new createError.BadRequest());
```

Comment:
You should probably send a message with this error. You almost always want to send an error message.